cmd/controller-manager/app/controller-manager.go

```diff
@@ -190,10 +193,13 @@ func getFederationConfig(opts *options.Options) *corev1a1.FederationConfig {
 		}
 
 		err := client.Get(context.Background(), fedConfig, namespace, name)
-		if err != nil {
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
This statement will be true for any error that isn't `IsNotFound`.  Likely `IsNotFound` should trigger this code path instead.

Reply:
Obvious logic error. Thanks!